aws/table_aws_route53_zone.go

```diff
@@ -71,13 +72,22 @@ func tableAwsRoute53Zone(_ context.Context) *plugin.Table {
 				Description: "The number of resource record sets in the hosted zone.",
 				Type:        proto.ColumnType_INT,
 			},
+			{
+				Name:        "query_logging_configs",
+				Description: "A list of configuration for DNS query logging that is associated with the current AWS account.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getHostedZoneQueryLoggingConfigs,
+				Transform:   transform.FromField("QueryLoggingConfigs"),
```

Comment:
Do we need the transformation here since the column name matches the field name?